tests/components/zwave_mqtt/test_scenes.py

```diff
@@ -0,0 +1,88 @@
+"""Test Z-Wave (central) Scenes."""
+from .common import MQTTMessage, setup_zwave
+
+from tests.common import async_capture_events
+
+
+async def test_scenes(hass, generic_data, sent_messages):
+    """Test setting up config entry."""
+
+    receive_message = await setup_zwave(hass, fixture=generic_data)
+    events = async_capture_events(hass, "zwave_mqtt.scene_activated")
+
+    # Publish fake scene event on mqtt
+    message = MQTTMessage(
+        topic="OpenZWave/1/node/39/instance/1/commandclass/43/value/562950622511127/",
+        payload={
+            "Label": "Scene",
+            "Value": 16,
+            "Units": "",
+            "Min": -2147483648,
+            "Max": 2147483647,
+            "Type": "Int",
+            "Instance": 1,
+            "CommandClass": "COMMAND_CLASS_SCENE_ACTIVATION",
+            "Index": 0,
+            "Node": 7,
+            "Genre": "User",
+            "Help": "",
+            "ValueIDKey": 122339347,
+            "ReadOnly": False,
+            "WriteOnly": False,
+            "ValueSet": False,
+            "ValuePolled": False,
+            "ChangeVerified": False,
+            "Event": "valueChanged",
+            "TimeStamp": 1579630367,
+        },
+    )
+    message.encode()
+    receive_message(message)
+    # wait for the event
+    await hass.async_block_till_done()
+    assert len(events) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5460460)